src/components/Login.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import LoginLocal from './LoginLocal'
+import RegisterNewUser from './RegisterNewUser'
+export default class Login extends React.Component{
```

Comment:
The linter I think catches this, but it's worth noting here: No need to do all this `class` and `React.Component` in es6 anymore.  Look at [this StackOverflow link](http://stackoverflow.com/a/40853268) to see how you can do this in es6.